src/playercards/CardsThatSealTokens.ttslua

```diff
@@ -44,25 +44,23 @@ Example 1: Crystalline Elder Sign
 This card can only seal the "+1" or "Elder Sign" token,
 it does not need specific options for multi-sealing or releasing.
 Thus it should be implemented like this:
-
-> VALID_TOKENS = {
->   ["+1"] = true,
->   ["Elder Sign"] = true
-> }
-> require ("playercards/CardsThatSealTokens") -- includes a space after "require" to not executing bundling
+  > VALID_TOKENS = {
+  >   ["+1"] = true,
+  >   ["Elder Sign"] = true
+  > }
+  > require ("playercards/CardsThatSealTokens") -- includes a space after "require" to not executing bundling
```

Comment:
I know this was there before, but reading it again after some time the comment here is confusing.  It describes why the space is there but isn't clear whether it should be in the actual code.

IMHO the require line isn't necessary in each example.  You already explain up top that this should be required after the data definitions.  The examples are for the data definitions, the require never changes.

If you feel strongly that it helps maybe change it to something like:
> require...

That would stop it from hitting the bundling regex and remove the need for the explainer